boot/makebootable.go

```diff
@@ -186,12 +202,7 @@ func makeBootable20(model *asserts.Model, rootdir string, bootWith *BootableSet)
 			return fmt.Errorf("cannot extract recovery system kernel assets: %v", err)
 		}
 
-		// record which recovery system is to be used
-		// TODO:UC20: should we do this for all bootloaders?
-		whereRecoveryEnv := map[string]string{
-			"snapd_recovery_system": bootWith.RecoverySystemLabel,
-		}
-		if err := bl.SetBootVars(whereRecoveryEnv); err != nil {
+		if err := bl.SetBootVars(blVars); err != nil {
```

Comment:
I think this is just redoing what we did above and can go away now?

Reply:
ah yes I missed this, you're right